vendor/composer/autoload_commands_classmap.php

```diff
@@ -6,4 +6,100 @@
 $baseDir = dirname($vendorDir);
 
 return array(
+    'CLI_Command' => $vendorDir . '/wp-cli/wp-cli/php/commands/src/CLI_Command.php', 
```

Comment:
If you commit changes to the vendor / composer stuff, be sure to run the `composer install --no-dev` first to prevent extra autoloader classes from slipping through.